### PR TITLE
Modernizing the project

### PR DESCRIPTION
**Pull request by carterscottm**
I've been working on this PR off and on for a while to update this repo.  I've added several additional color schemes from the wiki, and removed the western_rust color scheme, which was apparently incomplete and caused issues for some users.

I also removed some scripts from  the extras directory, as more recent versions of DFHack have rendered them obsolete and/or incompatible.  Some whitespace editing/reordering also took place with some of the embark profiles.

### CODE REVIEW

**Comment by PeridexisErrant** [comment]
Looks good to me - thanks!

**Comment by carterscottm** [comment]
Thanks @PeridexisErrant.  I didn't realize until I submitted this PR that your PR from August is still open, and some of the changes you proposed are in my PR as well.  Will merging one of them cause problems with the other one?

**Comment by PeridexisErrant** [comment]
Gosh, I'd completely forgotten about it to be honest.  If you see anything there that you think should be copied over, feel free to do so - and then just close the old pull, it's probably out of date anyway!

File: PyLNP.json [hunk 1]
```diff
@@ -46,7 +47,7 @@
         },
         "Other Automation plugins": {
             "command": "multicmd autonestbox start; repeat -time 7 -timeUnits days -command [ burial ] -name burial",
-            "tooltip": "Enables the autonestbox and burial scripts.",
+             "tooltip": "Enables the autonestbox and burial scripts.",
```
Comment (lethosor) [inline PyLNP.json]: Was this intentional?

Comment (carterscottm) [inline PyLNP.json]: No, that's just a typo.  I'll amend my last commit to fix this one.

**Comment by carterscottm** [comment]
I've added a few more tilesets and fixed a missing bracket in PyLNP.json.  If nobody objects in the next few days, I'll go ahead and squash/merge the changes, unless somebody else wants to do the merge.

**Pull request merged**